### PR TITLE
support input mask for embedded and merge layers.

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

File: keras2onnx/ke2onnx/embedding.py [hunk 1]
```diff
@@ -3,16 +3,21 @@
...
 
 
 def convert_keras_embed(scope, operator, container):
     op = operator.raw_operator  # Keras Embedding layer object
-    if hasattr(op, 'mask_zero') and op.mask_zero == True:
-        raise NotImplementedError("Embedding layer mask_zero attribute cannot be converted")
+    #  if mask_zero specified, the output_mask tensor needed by calculated
+    if hasattr(op, 'mask_zero') and op.mask_zero is True:
+        oopb = OnnxOperatorBuilder(container, scope)
+        # output_mask = K.not_equal(inputs, 0)
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/embedding.py]: remove this line?

File: keras2onnx/ke2onnx/merge.py [hunk 2]
```diff
@@ -36,3 +36,17 @@ def convert_keras_merge_layer(scope, operator, container):
             # Keep accumulate changes through iterations using buffer tensors
             intermediate_tensor_name = scope.get_unique_variable_name('intermediate_tensor')
         apply_merge_operation(scope, [left_tensor_name, right_tensor_name], intermediate_tensor_name, container)
+
+    if operator.output_masks:
+        #    masks = [array_ops.expand_dims(m, axis=0) for m in mask if m is not None]
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/merge.py]: do we need these commented lines?

Comment (wenbingl) [reply keras2onnx/ke2onnx/merge.py]: same above

File: keras2onnx/ke2onnx/embedding.py (hunk 1 above)
Comment (wenbingl) [reply keras2onnx/ke2onnx/embedding.py]: It's how Keras calc the mask.

**Pull request merged**